app/component/map/tile-layer/TileLayerContainer.js

```diff
@@ -45,13 +46,23 @@ import TileContainer from './TileContainer';
 import Loading from '../../Loading';
 import { isFeatureLayerEnabled } from '../../../util/mapLayerUtils';
 import MapLayerStore, { mapLayerShape } from '../../../store/MapLayerStore';
+import EcoCounterPopup from '../popups/EcoCounterPopup';
+import { isBrowser } from '../../../util/browser';
 
 const initialState = {
   selectableTargets: undefined,
   coords: undefined,
   showSpinner: true,
 };
 
+// TODO: Add other domains when site data is available.
+const getSiteIdFromChannelId = (id, domain) => {
+  if (domain === 'Oulu_kaupunki') {
+    return `${id.slice(0, 2)}0${id.slice(3, id.length)}`;
+  }
```

Comment:
I'm assuming here that the third character of the`siteId` in 'Oulu_kaupunki' domains always denotes the different direction and user types, and that 0 is always the "base" site. I'm not sure if this is always the case. Also do the other domain siteIds work similarly?

Reply:
There is no guarantee that this is the case. The "user type", i.e. pedestrian or bicycle is contained in it's own property `userType` for each channel. The direction however is not specified directly anywhere.

Reply:
After reviewing the documentation at http://eco-test2.com/apidoc/wso2/apidoc.html, it says the `sens` property includes the direction; "Direction detection information. {1: IN direction, 2: OUT direction, 5: No direction distinction}".